LocalMultiplayerAgent.UnitTest/DeploymentTests.cs

```diff
@@ -0,0 +1,119 @@
+﻿using FluentAssertions;
+using Microsoft.Azure.Gaming.LocalMultiplayerAgent.Config;
+using Microsoft.Azure.Gaming.LocalMultiplayerAgent.DeploymentTool;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using Moq;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+/*using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;*/
+
+namespace Microsoft.Azure.Gaming.LocalMultiplayerAgent.UnitTests
+{
+    [TestClass]
+    internal class DeploymentTests
```

Comment:
nit: This is unit test for DeploymentSettings so we can rename it to DeploymentSettingsTests so not confused with DeploymentScriptTests